### ISSUE TITLE
Improve test suite

### ISSUE THREAD

**Issue by arejula27** [Vanilla-OS/Vib#10]
At present, the project lacks sufficient tests. It would be beneficial to develop a concise test suite, particularly for the core package.

**Comment by kbdharun** [comment]
Yep, I think we can define a Sample recipe along with the intended Containerfile with support for all Vib syntax, we can implement test logic so that the test passes whenever the Containerfile is generated as expected without additional issues/modifications.
